src/main/java/org/kohsuke/github/GHBranch.java

```diff
@@ -118,7 +118,7 @@ public String getSHA1() {
      *             if disabling protection fails
      */
     public void disableProtection() throws IOException {
-        root.createRequest().method("DELETE").withUrlPath(protection_url).send();
+        root.createRequest().method("DELETE").setRawUrlPath(protection_url).send();
```

Comment:
@bitwiseman protection_url is a full url `http://localhost:36413/repos/github-api-test-org/temp-testDisableProtectionOnly/branches/master/protection`, so I changed it , is this similar to what you have in mind

Reply:
I will add the wire mock snapshot soon.

Reply:
Yes, this is the kind of change that I'm looking for.